queries.sql

```diff
@@ -1,3 +1,96 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT * FROM animals WHERE neutered = true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* Start transaction */
+
+BEGIN;
+
+/* update transaction */
+
+UPDATE animals
+SET species = 'unspecified';
+
+SELECT * FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
+
+/* Start transaction */
+
+BEGIN;
+
+/* update transaction */
+
+UPDATE animals
+SET species = 'digimon' WHERE name LIKE '%mon';
+
+SELECT * FROM animals;
+
+UPDATE animals 
+SET species = 'pokemon' WHERE species IS NULL;
+
+SELECT * FROM animals;
+
+/* Commit the transaction */
+COMMIT;
+SELECT * FROM animals;
+
+/* Start transaction */
+
+BEGIN;
+/* Delete a transaction */
+DELETE FROM animals;
+/* Return to previuse state of transation */
+ROLLBACK;
+SELECT * FROM animals;
+
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
```

Comment:
- [ ] Make sure to  Start the transaction before you `DELETE` all animals born after Jan 1st, 2022 . Eg:

```sql
    BEGIN;

    DELETE FROM animals WHERE  date_of_birth > '2022/01/01' ;
```